mppm/player-tags/player-tags-create.md

```diff
@@ -4,11 +4,11 @@ title: Create a Player Tag
 description: Use these steps to create Player Tags to configure the player actions in Multiplayer Play Mode.
 ---
 
-To create a **Player Tag**, perform mthe following actions: 
+To create a **Player Tag**, perform the following actions: 
 
-1. In the **Unity Editor**, open the **MPPM** window by navigating to **Window** > **Multiplayer Play Mode**.
-2. Select the plus sign ![plus sign](/img/add.png) under **Player Tags** to create your new tag.
+1. In the **Unity Editor**, open the **Multiplayer Play Mode** window by navigating to **Window** > **Multiplayer Play Mode**.
+2. Select the plus sign ![plus sign](/img/add.png) under **Player Tags** to create a new tag.
 3. Name the **Player Tag**.
-4. Select **Save**. Your new tag is automatically added to the `VirtualProjectsConfig.json` file in the `ProjectsSettings` folder.
-5. Your new tag should also appear under the **Player Tags** section of the **MPPM** window and in the dropdown menu for the **Tag** option of each **Player**.
+4. Select **Save**. The new tag is automatically added to the `VirtualProjectsConfig.json` file in the `ProjectsSettings` folder.
```

Comment:
Where is this folder? I was going to check that there's an "s" on the end of "Projects," but I don't know where to find the folder.

Reply:
Following this one up with Johann

Reply:
This is in the project directory, I'll update the instructions to state this :)
![image](https://github.com/Unity-Technologies/com.unity.multiplayer.docs/assets/63712500/4c57ac09-8b21-4c9d-aa7c-c6de93768cd3)